### PR TITLE
docs: deprecate SHA notice [DET-4336]

### PR DESCRIPTION
**Pull request by naren-determined**
## Description

Deprecation notice for `Adaptive (Simple)` and `Adaptive (Advanced)` searchers.

## Checklist

- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
See [Release Note](../docs/release-notes/README.md) for details.

[DET-1234]: https://determinedai.atlassian.net/browse/DET-1234

### CODE REVIEW

File: docs/reference/experiment-config.txt [hunk 1]
```diff
@@ -659,6 +659,8 @@ large-scale experiments with hundreds or thousands of trials.
 Adaptive (Simple)
 =================
 
+Note: To be deprecated. Recommended to use state-of-the-art Adaptive (ASHA).
```
Comment (neilconway) [inline docs/reference/experiment-config.txt:662]: Is it deprecated, or is it going to be deprecated?

> Recommended to use state-of-the-art Adaptive (ASHA).

Full sentences please :)

File: docs/topic-guides/hp-tuning-det/index.txt [hunk 2]
```diff
@@ -32,6 +32,9 @@ search experiments in the distributed setting.
 Synchronous Searchers
 =====================
 
+Note: Adaptive (Simple) and Adaptive (Advanced ) to be deprecated.
```
Comment (neilconway) [inline docs/topic-guides/hp-tuning-det/index.txt:35]: Whitespace.

File: docs/reference/experiment-config.txt (hunk 1 above)
Comment (naren-determined) [reply docs/reference/experiment-config.txt:662]: This is a notice that it is going to be deprecated

Comment (neilconway) [reply docs/reference/experiment-config.txt:662]: Why not just deprecate it now? i.e., to me, "deprecation" is itself a notice. A notice of an upcoming notice seems a little redundant, no?

File: docs/reference/experiment-config.txt [hunk 3]
```diff
@@ -659,6 +659,9 @@ large-scale experiments with hundreds or thousands of trials.
 Adaptive (Simple)
 =================
 
+Note: Adaptive (Simple) will be deprecated. We recommend users to use
```
Comment (liamcli) [inline docs/reference/experiment-config.txt:662]:
```suggestion
Note: Adaptive (Simple) will be deprecated. We recommend using
```

File: docs/reference/experiment-config.txt [hunk 4]
```diff
@@ -727,6 +730,9 @@ configure for most applications of hyperparameter search.
 Adaptive (Advanced)
 ===================
 
+Note: Adaptive (Advanced) will be deprecated. We recommend users to use
```
Comment (liamcli) [inline docs/reference/experiment-config.txt:733]:
```suggestion
Note: Adaptive (Advanced) will be deprecated. We recommend using
```

File: docs/topic-guides/hp-tuning-det/hp-adaptive-advanced.txt [hunk 5]
```diff
@@ -4,6 +4,9 @@
  Hyperparameter Search: Adaptive (Advanced)
 ############################################
 
+Note: Adaptive (Advanced) will be deprecated. We recommend users to use
```
Comment (liamcli) [inline docs/topic-guides/hp-tuning-det/hp-adaptive-advanced.txt:7]:
```suggestion
Note: Adaptive (Advanced) will be deprecated. We recommend using
```

File: docs/topic-guides/hp-tuning-det/hp-adaptive-simple.txt [hunk 6]
```diff
@@ -4,6 +4,9 @@
  Hyperparameter Search: Adaptive (Simple)
 ##########################################
 
+Note: Adaptive (Simple) will be deprecated. We recommend users to use
```
Comment (liamcli) [inline docs/topic-guides/hp-tuning-det/hp-adaptive-simple.txt:7]:
```suggestion
Note: Adaptive (Simple) will be deprecated. We recommend using
```

File: docs/topic-guides/hp-tuning-det/index.txt [hunk 7]
```diff
@@ -32,6 +32,9 @@ search experiments in the distributed setting.
 Synchronous Searchers
 =====================
 
+Note: Adaptive (Simple) and Adaptive (Advanced) to be deprecated.
```
Comment (liamcli) [inline docs/topic-guides/hp-tuning-det/index.txt:35]:
```suggestion
Note: Adaptive (Simple) and Adaptive (Advanced) will be deprecated.
```

File: docs/topic-guides/hp-tuning-det/index.txt [hunk 8]
```diff
@@ -32,6 +32,9 @@ search experiments in the distributed setting.
...
+Recommended to use state-of-the-art Adaptive (ASHA).
```
Comment (liamcli) [inline docs/topic-guides/hp-tuning-det/index.txt:36]:
```suggestion
We recommend using the state-of-the-art Adaptive (ASHA) searcher.
```

**Review by liamcli — COMMENTED** [review]
Very minor changes.

File: docs/reference/experiment-config.txt [hunk 9]
```diff
@@ -659,8 +659,8 @@ large-scale experiments with hundreds or thousands of trials.
 Adaptive (Simple)
 =================
 
-Note: Adaptive (Simple) will be deprecated. We recommend using
-the state-of-the-art Adaptive (ASHA) searcher.
+Note: Adaptive (Simple) will be deprecated. We recommend using the
```
Comment (justin-determined-ai) [inline docs/reference/experiment-config.txt:662]: question: Are there any treatments we can use to make this more prominent? Something like the screenshot below, with a more warning-y color?

Screen Shot 2020-10-26 at 15 51 52

Comment (justin-determined-ai) [reply docs/reference/experiment-config.txt:662]: Aha, literally answered my own question by scrolling down a tiny bit :) Can we use this instead, with a link to the topic guide for ASHA?

Screen Shot 2020-10-26 at 15 52 59

Comment (naren-determined) [reply docs/reference/experiment-config.txt:662]: Ooh yeah let me add this

**Review by justin-determined-ai — APPROVED** [review]
LGTM! I think it's actually _more_ accurate to say that these searchers "[...] are now deprecated and will be removed in a future release." I think if you change that wording this is good to go!

(Using https://en.wikipedia.org/wiki/Deprecation as a definition.)

**Pull request merged**